lib/git_reflow/git_server/git_hub/pull_request.rb

```diff
@@ -63,7 +63,11 @@ def comments
         end
 
         def last_comment
-          "#{comments.last.body.inspect}"
+          if comments.nil? or comments.last.nil?
```

Comment:
@simonzhu24 If `comments` always returns an array then the `comments.nil?` check is not needed, and should be removed.  The only test we need is `comments.last.nil?`

Regarding @codenamev's original comment, `last_comment` should always return a string.  When there is no last comment it will be an empty string.